src/main/kotlin/com/cognifide/gradle/aem/environment/Environment.kt

```diff
@@ -205,7 +212,7 @@ class Environment(val aem: AemExtension) {
     /**
      * Defines hosts to be appended to system specific hosts file.
      */
-    fun hosts(values: Iterable<String>) = hosts.define(values)
+    fun hosts(names: Iterable<String>) = hosts.define(dockerType.hostIp, names)
```

Comment:
magic factory method which will add correct hosts entries in case of installed type of Docker